### PR TITLE
Accept a comma-separated list of messages IDs in ``--help-msg``

### LINKED ISSUE #7471: --help-msg no longer accepts comma separate messages since v2.14.0
I'm not sure if this is intentional since I don't find it mentioned anywhere in the changelog.

In v2.13.9:

```
$ pylint --help-msg C0102,C0103
:disallowed-name (C0104): *Disallowed name "%s"*
  Used when the name matches bad-names or bad-names-rgxs- (unauthorized names).
  This message belongs to the basic checker.

:invalid-name (C0103): *%s name "%s" doesn't conform to %s*
  Used when the name doesn't conform to naming rules associated to its type
  (constant, variable, class...). This message belongs to the basic checker.
```

In v2.14.0:

```
$ pylint --help-msg C0102,C0103
No such message id or symbol 'C0102,C0103'.
```

Possibly a side effect when migrating to `argparse`?

I don't think this is a big deal, and not sure if there are other similar affected flags. But it might be still worth mentioning somewhere about this change.

### PR DESCRIPTION
**Pull request by DanielNoord**
- [x] Write a good description on what the PR does.
- [x] Create a news fragment with `towncrier create <IssueNumber>.<type>` which will be
  included in the changelog. `<type>` can be one of: new_check, removed_check, extension,
  false_positive, false_negative, bugfix, other, internal. If necessary you can write
  details or offer examples on how the new change is supposed to work.
- [x] If you used multiple emails or multiple names when contributing, add your mails
      and preferred name in ``script/.contributors_aliases.json``

## Type of Changes

|     | Type                   |
| --- | ---------------------- |
| ✓   | :bug: Bug fix          |

## Description

Closes #7471.

### CODE REVIEW

**Review by Pierre-Sassoulas — COMMENTED** [review]
Looks pretty straightforward. I would have thought that the parsing would be done by the same function that disable/enable (to handle "all") but it works and it's simple.

File: pylint/config/callback_actions.py [hunk 1]
```diff
@@ -158,6 +158,8 @@ def __call__(
         option_string: str | None = "--help-msg",
     ) -> None:
         assert isinstance(values, (list, tuple))
+        assert isinstance(values[0], str)
```
[the remark this answers is not in the document]
Comment (DanielNoord) [reply pylint/config/callback_actions.py]: The `typing` comes from `argparse`. I believe `mypy` starts complaining if we narrow down in the signature.

**Comment by DanielNoord** [comment]
> Looks pretty straightforward. I would have thought that the parsing would be done by the same function that disable/enable (to handle "all") but it works and it's simple.

That function isn't really accessible. I changed it to use the `csv` checker though.

**Review by Pierre-Sassoulas — APPROVED** [review]
👌

**Pull request merged**